index.js

```diff
@@ -420,14 +427,15 @@ function handleRequests() {
 /**
  * @type { typeof import('./index').init }
  */
-function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 1000 * 10, aSwaggerUiServePath = 'api-docs', aMongooseModels = [], aTags = undefined) {
+function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 1000 * 10, aSwaggerUiServePath = DEFAULT_SWAGGER_UI_SERVE_PATH, aMongooseModels = [], aTags = undefined, aIgnoredNodeEnvironments = DEFAULT_IGNORE_NODE_ENVIRONMENTS) {
   handleResponses(aApp, {
```

Comment:
Shall we print something like "WARNING!!! process.env.NODE_ENV is not defined. To disable the module set process.env.NODE_ENV to 'production'" in case `process.env.NODE_ENV` is undefined?

Reply:
Or actually "... to one of `ignoredNodeEnvironments`"

Reply:
Just included `winston` to log a warning when process.env.NODE_ENV is undefined.